### DISCUSSION TITLE
Z-index on create-gui-menu-drop-down?

### DISCUSSION THREAD

**Discussion by sabracrolleton** [rabbibotton/clog#134] [Q&A] [answered]
I combined Tutorial 22 (clog-gui-menu-bar) with the create-panel-box-layout, using Tutorial 27 as a reference, using create-div instead of create-img for the center-panel.

If (create-gui-menu-bar ...) references either head or body, you see the menu bar, but cannot see the drop down menus. They are visible if (create-gui-menu-bar ...) references the center-panel, but obviously that changes the horizontal position of the menu.

Does it have to do with a z-index I need to set somewhere or am I missing something else?

**Comment by rabbibotton** [comment, marked as answer]
It is a z-index issue. clog-gui is intended to be used with clog-gui-window's and doesn't play so nice when trying to play on the desktop. If you are not looking to use the clog-gui-window's just use clog-web instead that also has the menu bar.

**Reply by sabracrolleton** [reply]
Thank you.
